analytics-integrations/localytics/src/main/java/com/segment/analytics/internal/integrations/LocalyticsIntegration.java

```diff
@@ -128,7 +128,14 @@
     super.track(track);
     Properties props = track.properties();
     setContext(track.context());
-    Localytics.tagEvent(track.event(), props.toStringMap());
+
+    final long revenue = (long) (props.revenue() * 100);
```

Comment:
I think it's better to check on a long than a double.  For instance, what happens if revenue gets passed in as 0.005, or half a cent?  It will pass the condition but then get truncated and tracked as zero.  Also, it should take about a nanosecond to multiply, you don't save anything you can measure.